Integrations/integration-SymantecMSS.yml

```diff
@@ -0,0 +1,506 @@
+            "Description": result["Description"],
+            "Analyst Assessment": result["AnalystAssessment"],
+            "Number of Analyzed Signatures": result["NumberOfAnalyzedSignatures"],
+            "Signaturtes": json.dumps(sigs) or "",
+            "Related Incidents": json.dumps(result["RelatedIncidents"]["IncidentNumber"]) or "",
+            "Comment": result["IncidentComments"]["IncidentComment"]["Comment"] or ""
+        }]
+        headers = [
+            "Incident Number",
+            "Time Created",
+            "Status",
+            "Classification",
+            "Assigned Person",
+            "Description",
+            "Analyst Assessment",
+            "Number of Analyzed Signatures",
+            "Signaturtes",
+            "Related Incidents",
+            "Comment"
+        ]
+        hr = tableToMarkdown("Incident query", flattenRelevantFields, headers)
+
+        # Set context
+        resultCtx = {
+            "IncidentNumber": result["IncidentNumber"],
+            "NumberOfAnalyzedSignatures": result["NumberOfAnalyzedSignatures"],
+            "SignatureList": {
+                "Signature": sigs
+            },
+            "TimeCreated": result["TimeCreated"],
+            "Classification": result["Classification"],
+            "Description": result["Description"],
+            "AnalystAssessment": result["AnalystAssessment"],
+            "CountryCode": result["CountryCode"],
+            "CountryName": result["CountryName"],
+            "RelatedTickets": result["RelatedTickets"],
+            "WorkFlowDetail": {
+                "Status": result["WorkFlowDetail"]["Status"],
+                "AssignedPerson": result["WorkFlowDetail"]["AssignedPerson"]
+            },
+            "IncidentComments": {
+                "IncidentComment": { # array?
+                    "CommentedTimeStampGMT": result["IncidentComments"]["IncidentComment"]["CommentedTimeStampGMT"],
+                    "Comment": result["IncidentComments"]["IncidentComment"]["Comment"],
+                    "CommentedBy": result["IncidentComments"]["IncidentComment"]["CommentedBy"]
+                }
+            },
+            "IncidentAttachmentItems": {
+                "IncidentAttachmentItem": { # array?
+                    "AttachmentNumber": result["IncidentAttachmentItems"]["IncidentAttachmentItem"]["AttachmentNumber"],
+                    "AttachmentName": result["IncidentAttachmentItems"]["IncidentAttachmentItem"]["AttachmentName"],
+                    "UploadDateGMT": result["IncidentAttachmentItems"]["IncidentAttachmentItem"]["UploadDateGMT"],
+                    "UploadBy": result["IncidentAttachmentItems"]["IncidentAttachmentItem"]["UploadBy"],
+                    "Comment": result["IncidentAttachmentItems"]["IncidentAttachmentItem"]["Comment"]
+                }
+            },
+            "RelatedIncidents": {
+                "IncidentNumber": result["RelatedIncidents"]["IncidentNumber"] # array for sure
```

Comment:
if you don't sure, you always can use `dq`. `dq(result, 'RelatedIncidents. IncidentNumber')`